validator/specs/lib/specUtility.js

```diff
@@ -37,7 +38,7 @@ function createB3dm(options) {
         batchTableBinary = Buffer.concat([batchTableJsonBuffer, Buffer.alloc(1)]);
     }
     if (options.unalignedByteLength) {
-        glb = Buffer.concat([glb, Buffer.alloc(1)]);
+        glb = Buffer.concat([glb, createGlb()]);
```

Comment:
This change isn't needed, this area is purposefully adding an extra byte to throw off byte alignment (tested in `validateCmpt`.

I believe the tests are still passing because the glb buffer is 12 bytes and therefore still doesn't meet the 8-byte alignment requirement.